### PR TITLE
Update input card styling

### PR DESCRIPTION
**Pull request by hdavidzhu**
image

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -39,6 +39,9 @@
     "webpack-dev-server": "^3.1.4"
   },
   "dependencies": {
+    "@fortawesome/fontawesome-svg-core": "^1.2.22",
+    "@fortawesome/free-solid-svg-icons": "^5.10.2",
+    "@fortawesome/react-fontawesome": "^0.1.4",
```
Comment (jgozal) [inline package.json]: great that we have these now

File: src/components/main-form.js [hunk 2]
```diff
@@ -217,199 +173,275 @@ class MainForm extends React.Component {
...
                   />
                   <InputGroupAddon addonType='append'>
                     <InputGroupText>%</InputGroupText>
                   </InputGroupAddon>
                 </InputGroup>
-              </InputWrapper>
-              <InputWrapper>
-                <label>Mortgage Option 2</label>
+                <Accordion title='How do I invest?'>
+                  {/* TODO: Could this be Vanguard instead? */}
```
Comment (jgozal) [inline src/components/main-form.js]: Please change it to Vanguard in view of the fb poll results

**Comment by jgozal** [comment]
> Code looks good to me. Section titles and subtitles look a little large in proportion to the expandable bullet points. Think you can fix it up a bit if you agree? Also, will you be opening a separate PR for the tooltips?

File: src/components/main-form.js (hunk 2 above)
Comment (hdavidzhu) [reply src/components/main-form.js]: 😄 oh right, good catch

**Comment by hdavidzhu** [comment]
@jgozal Yeah I'll make another PR just for the tool tips. Some feedback we got was that the dropdowns (at least when they were blue links) looked too visible, so I was trying to down play it so people can focus on the input

**Comment by jgozal** [comment]
I think we just need to make the subtitles a bit smaller and thing will look better. I don't think we should make the expandable bullet points bigger.



> @jgozal Yeah I'll make another PR just for the tool tips. Some feedback we got was that the dropdowns (at least when they were blue links) looked too visible, so I was trying to down play it so people can focus on the input

**Comment by jgozal** [comment]
@hdavidzhu do you think it would be a good idea to automatically close all expandable bullet points when one opens?

File: src/components/main-form.js [hunk 3]
```diff
@@ -217,199 +173,275 @@ class MainForm extends React.Component {
...
+                  15-year versus 30-year mortgages.
+                </Accordion>
+                <Accordion
+                  title={`What happens when after you’re done paying off the ${_.min(
+                    [option1.term, option2.term]
+                  )}-year mortgage?`}
+                >
+                  {/* TODO: Convert to dynamic numbers */}
+                  This tool assumes that you’ll invest the difference between
+                  the payment of the 15-year and the 30-year mortgages. If you
```
Comment (jgozal) [inline src/components/main-form.js]: @hdavidzhu can you make all mentions of 15-year, 30-year dynamic? Completely forgot about this

File: src/components/main-form.js [hunk 4]
```diff
@@ -217,199 +173,275 @@ class MainForm extends React.Component {
...
+                  <a href='https://www.khanacademy.org/economics-finance-domain/core-finance/inflation-tutorial/inflation-basics-tutorial/v/inflation-data'>
+                    Khan Academy
+                  </a>{' '}
+                  has a brief video that talks about the data behind the CPI.
+                </Accordion>
+                <Accordion title='How inflation affects your mortgage and investments'>
+                  As time passes, inflation makes your mortgage payments cheaper
+                  and your return on investment lower. We factor all of this
+                  automatically so the final result you see on your right is not
+                  the actual money you will have gained/lost in 30 years, but
```
Comment (jgozal) [inline src/components/main-form.js]: 30 year mentions need to be dynamic

File: src/components/main-form.js [hunk 5]
```diff
@@ -292,199 +250,275 @@ class MainForm extends React.Component {
...
+                </Accordion>
+                <Accordion title='Long Term vs. Short Term Investments'>
+                  Long term investments are those you hold for longer than a
+                  year. In contrast, short term investments are usually held for
+                  a year or less. Note that short term investments usually have
+                  a <b>higher tax burden</b> than long term investments.{' '}
+                  <a href='https://www.edwardjones.ca/financial-focus/investment-topics/short-term-vs-long-term-investments.html'>
+                    Edward Jones
+                  </a>
+                  provides some info about the distinctions between a long term
```
Comment (jgozal) [inline src/components/main-form.js]: seems like we need a space before 'provides'

File: src/components/accordion.js [hunk 6]
```diff
@@ -20,17 +20,19 @@ class Accordion extends Component {
...
           ) : (
             <small className='text-muted'>
-              <FontAwesomeIcon icon={faCaretRight} className='mr-1' />
+              <FontAwesomeIcon icon={faCaretRight} className='mr-2' />
               {this.props.title}
             </small>
           )}
         </a>
-        <Collapse isOpen={this.state.collapse}>{this.props.children}</Collapse>
+        <Collapse className='pl-3 py-2' isOpen={this.state.collapse}>
```
Comment (jgozal) [inline src/components/accordion.js]: is this padding here? whatever you did drastically improved the spacing within the expandable bullet points so it doesn't look as crammed anymore. Thank you

**Comment by hdavidzhu** [comment]
https://github.com/jgozal/moneycompass/issues/25

Captured in future work. Thanks for the suggestions!

**Pull request merged**